core/build.sbt

```diff
@@ -8,5 +8,6 @@ libraryDependencies ++= Seq(
   "redis.clients" % "jedis" % "2.7.2",
   Logging.logbackClassic,
   Logging.logbackCore,
-  Logging.slf4jApi
+  Logging.slf4jApi,
+  "org.apache.commons" % "commons-lang3" % "3.4"
```

Comment:
define this as a val in https://github.com/allenai/common/blob/4cf2a6194091cd4b596aa8f6db4e36f55d6e76f3/project/Dependencies.scala